File: platform/src/main/java/ru/abondin/hreasy/platform/service/DateTimeService.java
package ru.abondin.hreasy.platform.service;

import org.springframework.stereotype.Component;

import java.time.OffsetDateTime;

/**
 * Service to deal with dates and time.
 * Can be used to do some actions in the past or in the future if required
 */
@Component
public class DateTimeService {
    public OffsetDateTime now() {
        return OffsetDateTime.now();
    }
}

File: platform/src/main/kotlin/ru/abondin/hreasy/platform/service/DateTimeService.kt
package ru.abondin.hreasy.platform.service

import org.springframework.stereotype.Component
import java.time.OffsetDateTime

/**
 * Service to deal with dates and time.
 * Can be used to do some actions in the past or in the future if required
 */
@Component
class DateTimeService {
    fun now() = OffsetDateTime.now();
}